File: Java/Lista 3/evenOddPositiveNegative.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        float lista[] = new float[5];
        int pos = 0;
        int neg = 0;
        int p = 0;
        int i = 0;
        for (int n = 0; n < lista.length; n++){
            lista[n] = sc.nextFloat();
            if(lista[n] > 0) {
                pos++;
            } else if (lista[n] < 0){
                neg++;
            }
            if(lista[n]%2 == 0){
                p++;
            } else if (lista[n]%2 != 0){
                i++;
            }
        }
        System.out.println(p + " valor(es) par(es)");
        System.out.println(i + " valor(es) impar(es)");
        System.out.println(pos + " valor(es) positivo(s)");
        System.out.println(neg + " valor(es) negativo(s)");
    }
}

File: Python/Lista 3/evenOddPositiveNegative.py
p = i = pos = neg = 0
n = float(input())
if n%2 == 0:
    p +=1
elif n%2 != 0:
    i +=1
if n > 0:
    pos +=1
elif n < 0:
    neg +=1
for v in range (4):
    n = float(input())
    if n%2 == 0:
        p +=1
    elif n%2 != 0:
        i +=1
    if n > 0:
        pos +=1
    elif n < 0:
        neg +=1
print(f'''{p} valor(es) par(es)
{i} valor(es) impar(es)
{pos} valor(es) positivo(s)
{neg} valor(es) negativo(s)''')